docs/integrations/sources/linkedin-ads-migrations.md

```diff
@@ -1,5 +1,36 @@
 # LinkedIn Ads Migration Guide
 
+## Upgrading to 2.0.0
+
+Version 2.0.0 introduces changes in the primary key selected for all *-analytics streams (including custom ones).
```

Comment:
Add this also: This upgrade changes primary key for *-analytics streams from pivotValues[array of strings] to string_of_pivot_values[string].

Reply:
updated